app/services/acuant/acuant_client.rb

```diff
@@ -49,12 +49,16 @@ def post_images(front_image:, back_image:, selfie_image:,
       if results.success? && liveness_checking_enabled
         pii = results.pii_from_doc
         selfie_response = post_selfie(image: selfie_image, instance_id: instance_id)
-        Acuant::Responses::ResponseWithPii.new(selfie_response, pii)
+        Acuant::Responses::ResponseWithPii.new(
+          acuant_response: selfie_response,
+          pii: pii,
+          billed: results.result_code&.billed?,
```

Comment:
@stevegsa just wanted to check my understanding on this...the only time we call the `GetResults` API is inside of this API where we post all 3 ---- but that's the newer flow right? Do you know which endpoint in the older flow we want to be be checking for this result code in?

Reply:
Unfortunately it looks like somebody removed the code to add_cost on the results call.

Reply:
Actually it looks good.  Those results calls are in fetch_doc_auth_results_or_redirect_to_selfie

Reply:
OK so `back_image_step` needs cost info added?

Reply:
added in 558d5143d7c559bf6a93e4e3bb415f81b600e1e6